yarn-project/end-to-end/src/e2e_token_contract.test.ts

```diff
@@ -496,6 +496,31 @@ describe('e2e_token_contract', () => {
           );
         });
 
+        it('transfer on behalf of other after deleting approval', async () => {
+          const balance0 = await asset.methods.balance_of_private(accounts[0].address).view();
+          const amount = balance0 / 2n;
```

Comment:
Why are we dividing by 0 here?

Reply:
we aren't - we are dividing by the current balance

Reply:
sorry meant to say 2